backend/backend/api/routes.py

```diff
@@ -346,4 +353,18 @@ async def github_comment(repo, pr_number, request):
         with gr.Column():
             github_comment_output = gr.Textbox(label="Response")
     github_comment_button.click(github_comment, inputs=[
-                                repo_input, pr_number_input, request_input], outputs=github_comment_output)
+                                repo_input, pr_number_input, request_input],
+                                outputs=github_comment_output)
+
+    gr.Markdown("## GitHub PR Review")
+    with gr.Row():
+        with gr.Column():
+            repo_input = gr.Textbox(label="Enter the repository here")
+            pr_number_input = gr.Textbox(
+                label="Enter the pull request number here")
+            github_comment_button = gr.Button("Review PR")
```

Comment:
The `github_comment_button` is reused from the previous section, but it now triggers a different function (`github_pr`) with fewer inputs. This could lead to confusion or errors if the `github_pr` function expects the third input (`request_input`). Ensure that `github_pr` can handle the provided inputs correctly.